.github/workflows/deploy-eb.yml

```diff
@@ -0,0 +1,40 @@
+name: Deploy EB Prod
+
+on:
+  push:
+    branches:
+      - master
+  repository_dispatch:
+    types: manual-trigger
```

Comment:
do we have to remove the manual trigger part so that it deploys automatically on push to master? is this line just for testing purposing right now?

Reply:
I just kept this here because it's what we have in our deploy job on sero-can-webapp. I'll look into a bit more